pom.xml

```diff
@@ -530,6 +537,24 @@
                     <artifactId>maven-dependency-plugin</artifactId>
                     <version>${maven-dependency-plugin.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>buildnumber-maven-plugin</artifactId>
+                    <version>1.4</version>
```

Comment:
Can we move this into the version properties up above in the pom:
https://github.com/confluentinc/common/blob/3d5b2b91fe41b7f030447f2251a4b45dabe2db8b/pom.xml#L110